### DISCUSSION TITLE
Profile / My Discussions - comments are missing

### DISCUSSION THREAD

**Discussion by ClaudiaK21** [community/community#3427] [Discussions]
Hello everyone,

Under Profile /My Discussions / Commented there are comments missing I made in another creators thread.

How long does it take until they appear?

_Comments from my own created threads are displayed correctly._

**Comment by PranamBhat** [comment]
Hi @ClaudiaK21 

It might be due to inactivity or session timed out.

Just refresh the page and go to your profile > Your Discussions:


![image](https://user-images.githubusercontent.com/32120033/109156481-e6729200-7796-11eb-886b-bcf05c778dfb.png)


Click on "Created" or "Commented" whichever you want and you will see your activity accordingly.

**Reply by ClaudiaK21** [reply]
Thanks, yes thats what I did, several times via F5, Shift + Reload Button and also Ctrl + R.

Your previous comment is now displayed in my created list. Maybe I didn't explained it right.
I didn't mean comments made my own created discussions, but comments **I made** in discussions from other authors.

At least **I thought** they will be listed too, not only comments I have made in my own created ones.
When I go to the github board and type in the search commenter:username  I will get the discussion in the search list, but this seems to be a little laborious.

**Reply by ClaudiaK21** [reply]
oh, I see now that I maybe mean the **Reply to a comment**?
These are not listed.

Hm, so a Reply to a comment might be not the best choice for me ;-)

**Reply by PranamBhat** [reply]
@ClaudiaK21 Yup! Correct.

I have gone to my "**Profile**" > "**Your discussions**" and gave "**commenter:ClaudiaK21**". Here is the result:

![image](https://user-images.githubusercontent.com/32120033/109165605-bed4f700-77a1-11eb-9122-6039ffcf3514.png)

**Reply by ClaudiaK21** [reply]
yes, this is right, same what I see, but I did a "Reply" to a comment here, which is not in the list
https://github.com/wekan/wekan/issues?q=is%3Aissue+is%3Aopen+commenter%3AClaudiaK21

Now I know such "comments" are not tracked in the profile discussion list, I will bookmark them for review if I need to, or use the standard comments ;-)

**Reply by PranamBhat** [reply]
@ClaudiaK21 

This might be because of **Private Repository/Organization/Team**.

In that case, you can opt this filter under the same repository:

Go to [https://github.com/wekan/wekan/](https://github.com/wekan/wekan/)

Then apply filter like this:


![image](https://user-images.githubusercontent.com/32120033/109170839-293c6600-77a7-11eb-8311-18073c411636.png)

**Reply by PranamBhat** [reply]
@ClaudiaK21 Kindly mark this as "**Answered**" if I have a cleared your queries. Thank you.
